### PR TITLE
Content - Borders - Created example pages

### PR DESCRIPTION
**Pull request by SebastianBurke**
related to: WET-396

### CODE REVIEW

File: common/borders/borders-en.html [hunk 1]
```diff
@@ -0,0 +1,125 @@
...
+<h3>CSS Class</h3>
+<dl class="dl-horizontal">
+	<dt><code>.brdr-left</code></dt>
+	<dd>Set a border to the left of the element</dd>
+	<dt><code>.brdr-tp</code></dt>
+	<dd>Set a border on top of the element</dd>
+	<dt><code>.brdr-rght</code></dt>
+	<dd>Set a border to the right of the element</dd>
+	<dt><code>.brdr-bttm</code></dt>
+	<dd>Set a border below the element</dd>
```
Comment (GormFrank) [inline common/borders/borders-en.html:124]: You are missing the following classes: `.brdr-0` (to remove borders) and `.brdr-rds-0` (to remove border radius)

File: common/borders/borders-en.html [hunk 2]
```diff
@@ -0,0 +1,125 @@
...
+				confuse users, especially those with cognitive disabilities or
+				attention disorders.
+			</li>
+			<li>
+				Don't use borders as a substitute for proper semantic structure
+				and organization. Use appropriate HTML elements, headings, and
+				other structural elements to convey content hierarchy.
+			</li>
+		</ul>
+	</div>
```
Comment (GormFrank) [inline common/borders/borders-en.html:78]: Please remove the Correct uses and the Compliance point, this is out-of-scope for GCWeb :)

File: common/borders/borders-en.html [hunk 3]
```diff
@@ -0,0 +1,125 @@
...
+    "share": "true",
+}
+---
+
+<p>
+	Borders are commonly used to provide visual separation between elements,
+	highlight specific content, or create decorative effects. By adjusting the
+	border properties, you can customize the appearance of the borders to match
+	your desired design.&nbsp;
+</p>
```
Comment (GormFrank) [inline common/borders/borders-en.html:24]: Please remove this descriptive blurb, and instead bring to the top the list you created at the bottom of this page.

File: common/borders/borders-en.html [hunk 4]
```diff
@@ -0,0 +1,125 @@
...
+				<p class="brdr-tp">
+                    Example 2
+				</p>
+				<p class="brdr-rght">
+                    Example 3
+				</p>
+			
+				<p class="brdr-bttm">
+                    Example 4
+				</p>
```
Comment (GormFrank) [inline common/borders/borders-en.html:95]: You are missing two examples, one for `.brdr-0` and one for `.brdr-rds-0`

File: common/borders/borders-en.html [hunk 5]
```diff
@@ -0,0 +1,125 @@
...
+				<h3 class="mrgn-tp-0">Code</h3>
+                <pre><code>// Example 1:
+                    &lt;p <strong>class=&quot;brdr-lft&quot;</strong>&gt;...&lt;/p&gt;</code></pre>          
+				<pre><code>// Example 2:
+                    &lt;p <strong>class=&quot;brdr-tp&quot;</strong>&gt;...&lt;/p&gt;</code></pre>
+                <pre><code>// Example 3:
+                    &lt;p <strong>class=&quot;brdr-rght&quot;</strong>&gt;...&lt;/p&gt;</code></pre>
+      
+                <pre><code>// Example 4:
+                    &lt;p <strong>class=&quot;brdr-bttm&quot;</strong>&gt;...&lt;/p&gt;</code></pre>
```
Comment (GormFrank) [inline common/borders/borders-en.html:109]: Don't forget to add the code samples for the 2 missing examples.

**Review by GormFrank — CHANGES_REQUESTED** [review]
Thank you for your contribution @SebastianBurke 🥇 

**IMPORTANT:** All of my requested changes apply to French page as well.

File: common/borders/borders-en.html [hunk 6]
```diff
@@ -0,0 +1,77 @@
...
+			"link": "https://wet-boew.github.io/themes-dist/GCWeb/index-en.html"
+		}
+	],
+	"secondlevel": false,
+	"dateModified": "2023-06-20",
+	"share": "true"
+}
+---
+
+<h3>CSS Class</h3>
```
Comment (duboisp) [inline common/borders/borders-en.html:19]: This create an issue with the page structure when jekyl generate the HTML final page. The heading level 2 is missing. 
Heading should always be in sequence.

Considering this page is short, I suggest to simply use a paragraph here.

```suggestion
<p>CSS Class</p>
```

File: common/borders/borders-en.html [hunk 7]
```diff
@@ -0,0 +1,77 @@
...
+	<dt><code>.brdr-0</code></dt>
+	<dd>Remove borders</dd>
+	<dt><code>.brdr-rds</code></dt>
+	<dd>Remove border radius</dd>
+</dl>
+<div class="row">
+	<div class="col-md-6">
+		<div class="panel panel-default">
+			<div class="panel-body">
+				<h3 class="mrgn-tp-0">Appearance</h3>
```
Comment (duboisp) [inline common/borders/borders-en.html:38]: Considering the page start with an h1 (heading level 1) created by jekyl by using the "title" property in your front-matter. This heading would need to be changed for an heading level 2.

```suggestion
				<h2>Appearance</h2>
```

(No need to remove the top margin here. FYI - If it was used in panel, then the panel title must be moved into the panel heading section.)

File: common/borders/borders-en.html [hunk 8]
```diff
@@ -0,0 +1,77 @@
...
+					Example 3
+				</p>
+				<p class="brdr-bttm">
+					Example 4
+				</p>
+				<p class="brdr-0">
+					Example 5
+				</p>
+				<p class="brdr-rds-0">
+					Example 6
```
Comment (duboisp) [inline common/borders/borders-en.html:55]: Be more descriptive with the text. Include the expected style. 

```suggestion
					Example of a left border
				</p>
				<p class="brdr-tp">
					Example of a top border
				</p>
				<p class="brdr-rght">
					Example 3
				</p>
				<p class="brdr-bttm">
					Example 4
				</p>
				<p class="brdr-0">
					Example 5
				</p>
				<p class="brdr-rds-0">
					Example 6
```

File: common/borders/borders-fr.html [hunk 9]
```diff
@@ -0,0 +1,77 @@
...
+				</p>
+				<p class="brdr-0">
+					Exemple 5
+				</p>
+				<p class="brdr-rds-0">
+					Exemple 6
+				</p>
+			</div>
+		</div>
+		<div class="panel panel-default">
```
Comment (duboisp) [inline common/borders/borders-fr.html:59]: I think the intend was to have it side by side so this is missing

```suggestion
		</div>
	</div>
	<div class="col-md-6">
		<div class="panel panel-default">
```

Note: I am assuming that you are also removing the panel for the code sample, I kept it here to show how to configure the Grid

**Review by duboisp — CHANGES_REQUESTED** [review]
There is an issue with your commit, you will need to remove the "merge" flag. The simplest will be to recreate the commit and then update your branch.

FYI  here the commit details
image

**Review by duboisp — CHANGES_REQUESTED** [review]

**Pull request closed without merging**